src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchUtil.java

```diff
@@ -24,6 +26,14 @@ static public String jsonPropertyToElasticProperty(String jsonProperty) {
 		
 	}
 	
+	static private void append (ArrayList<Reference> to, String ID, String baseURL)
+	{
+		Reference reference = new Reference();
+		reference.setId(ID + ".orex");
```

Comment:
Why the ".orex" at the end. This does not sound right.

Reply:
Done

Reply:
Done.